src/Simulation/EntryPointDriver.Tests/Tests.fs

```diff
@@ -565,6 +565,7 @@ let ``Submit uses default values`` () =
                Resource Group: myResourceGroup
                Workspace: myWorkspace
                Target: test.machine.noop
+               TargetCapability:
```

Comment:
We should have tests that check the basic handling of `TargetCapability`: that it uses the default value if the assembly constant if one is provided, and that the `--target-capability` option overrides that default.

Reply:
I'm working on the tests. In the meantime I would like to make sure I'm on the right track. Please review the current changes.